src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/LanguageHelper.cs

```diff
@@ -0,0 +1,33 @@
+using System.Globalization;
+
+namespace Altinn.Platform.Storage.Helpers
+{
+    /// <summary>
+    /// Helper methods for text.
+    /// </summary>
+    public static class LanguageHelper
+    {
+        /// <summary>
+        /// Checks if an language string is a valid two letter ISO name
+        /// </summary>
+        /// <param name="language">the language to check</param>
+        /// <returns>true if it is valid, false otherwise</returns>
+        public static bool IsTwoLetterISOName(string language)
+        {
+            if (language == null)
+            {
+                return false;
+            }
+
+            foreach(var culture in CultureInfo.GetCultures(CultureTypes.AllCultures))
+            {
```

Comment:
could simplify notation or at least to get rid of the foreach 
`return  CultureInfo.GetCultures.Contains(langugage) ? true : false`

Reply:
should we also ignore casing here to be on the safe side?

Reply:
Refactored, take a look. I prefer to be strict. If we are going with ISO standard, I feel that should be a requirement.

Reply:
looks good.